druid/src/contexts.rs

```diff
@@ -242,6 +276,60 @@ impl<'a> EventCtx<'a> {
         &self.window
     }
 
+    /// Create a new window.
+    /// `T` must represent the application state provided to [`AppLauncher::launch`].
+    ///
+    /// [`AppLauncher::launch`]: struct.AppLauncher.html#method.launch
+    pub fn new_window<T: Any>(&mut self, desc: WindowDesc<T>) {
+        if self.state_types.check_window_desc::<T>() {
+            self.submit_command(
+                Command::one_shot(commands::NEW_WINDOW, desc),
+                Target::Global,
+            );
+        } else {
+            const MSG: &str = "All application windows must represent the same type of state.";
```

Comment:
This message is confusing me.
Perhaps something like `WindowDesc<T> - T must match the application state.`